yarn.lock

```diff
@@ -4683,7 +4708,7 @@ class-utils@^0.3.5:
     isobject "^3.0.0"
     static-extend "^0.1.1"
 
-classnames@^2.2.5:
+classnames@^2.2.5, classnames@^2.2.6:
```

Comment:
Let's upgrade classnames to 2.2.6 so we are on the same patch universally (since Semantic UI also uses that version).

Reply:
These are dependencies, is there a way to upgrade another packages dependencies? I tried `yarn upgrade classnames` but it didn't alter the lockfile

Reply:
You're right. This is all good!